python/tox.ini

```diff
@@ -4,15 +4,22 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py36, py37, py38, py39
-
+envlist = {py36,py37,py38,py39}-{unit,integration}
 
 [testenv]
 deps =
-    pytest-mock
-    pillow
     pytest
-    pyyaml
     pytest-cov
+    pytest-mock
+    pyyaml
+
+[testenv:unit]
+commands =
+    pytest --junitxml={env:TOX_JUNIT_OUTPUT_DIR:../results/}/{env:TOX_JUNIT_OUTPUT_NAME:python.{envname}.xml} tests/rtl/
```

Comment:
The syntax here is `{env:ENV_VAR_NAME:default-value}`
I invented these `TOX_JUNIT_OUTPUT_{DIR,NAME}` env vars (copied from the jest stuff). CI will set them

Reply:
Makes sense